lib/index.js

```diff
@@ -927,7 +989,7 @@ Stream.prototype.pipe = function (dest) {
  * @api public
  */
 
-Stream.prototype.destroy = function () {
+addMethod('destroy', function () {
```

Comment:
Suggest removing the `{expose:true}` option and just make `addMethod` expose by default. Anything not exposed can be done via `Stream.prototype.method = function method()`. Especially since you've kept the internal methods under the `*.prototype` pattern.

Alternatively, keep the `addMethod` but drop the `{expose:true}`. We just expose *everything*, including `pull`, `write`, `fork`, etc.